WordPressPCL/WordPressPCL.xml

```diff
@@ -433,6 +425,13 @@
             <param name="httpHelper">reference to HttpHelper class for interaction with HTTP</param>
             <param name="defaultPath">path to site, EX. http://demo.com/wp-json/ </param>
         </member>
+        <member name="M:WordPressPCL.Client.PostStatuses.Get(System.Boolean,System.Boolean)">
+            <summary>
+            Get latest
+            </summary>
+            <param name="emb   <param name="httpHelper">reference to HttpHelper class for interaction with HTTP</param>
```

Comment:
Is not going to through a format error since there is no closing quote on: name="emb

Reply:
This file is autogenerated, in code all xml-doc comments printed correctly